magento/pwa-studio
# Checkout Items Review Component

## Description

Created the review of items in order component.

## Related Issue
Closes [PWA-406](https://jira.corp.magento.com/browse/PWA-406)

### Verification Steps
1. Add items to the cart.
2. Go to the checkout page.
3. Complete address and payment steps. You should see review of the items section before you place the order.

## Screenshots / Screen Captures (if appropriate)

**Desktop (Showing only 2):**

![image](https://user-images.githubusercontent.com/35203638/76900670-cbe62800-6867-11ea-811f-8167dcbaf9ad.png)

**Desktop (Showing All):**

![image](https://user-images.githubusercontent.com/35203638/76900692-d7395380-6867-11ea-8359-c5b24cfff33e.png)

**Mobile (Showing only 2):**

![image](https://user-images.githubusercontent.com/35203638/76900967-5464c880-6868-11ea-91d8-adf29b5cd871.png)

**Mobile (Showing All):**

![image](https://user-images.githubusercontent.com/35203638/76900882-2ed7bf00-6868-11ea-9ff6-2dfd6fdc3a72.png)


## Checklist
None
